.gitignore

```diff
@@ -61,3 +61,6 @@ tags
 
 # dotenv file
 .env
+
+# Local .vimrc
+.lvimrc
```

Comment:
Do you really need to exclude a file that used by only you? Could you put them into your own local gitignore?